appshell/appshell_extensions_mac.mm

```diff
@@ -1226,6 +1226,25 @@ int32 InstallCommandLineTools()
     AuthorizationRef authorizationRef = NULL;
     
     try {
+        
+        // Determine the location of the launch script.
+        // We have this file, brackets.sh, present inside resource folder.
+        
+
+        NSString* sourcePath;
+        
+        sourcePath = [[NSBundle mainBundle] pathForResource: @"brackets" ofType: @"sh"];
+        if(!sourcePath) {
+            // If we have not found this in the bundle try the hard path.
+            NSString* bundlePath = [[NSBundle mainBundle] bundlePath];
+            sourcePath = [bundlePath stringByAppendingString:@"/Contents/Resources/brackets.sh"];
+        }
+        
+        if(!sourcePath)
+            return ERR_FILE_NOT_FOUND;
```

Comment:
The documentation says `stringByAppendingString` could return `null in failure scenarios. So a check here would be better for sure.